circuit_knitting/cutting/cutting_evaluation.py

```diff
@@ -354,55 +350,6 @@ def _generate_cutting_experiments(
     return subexperiments_out, weights, subexperiments_legacy
 
 
-def _run_experiments_batch(
```

Comment:
No longer needed. Logic is now simplified in `execute_experiments` since we can just pass `SamplerResult` instances out